ui/imports/utils/Utils.qml

```diff
@@ -295,6 +295,15 @@ QtObject {
         return qsTr("%1D").arg(diffDay)
     }
 
+    function formatShortDate(value, isDDMMYYDateFormat) {
+        const formatDDMMYY = "d MMMM yyyy"
```

Comment:
Uhm.... :) OK, adding to my TODO as well

Reply:
@caybro have addressed all your comments :) Please review and approve when you can!